Taskfile.yml

```diff
@@ -12,6 +14,7 @@ tasks:
     cmds:
       - task: workspace
       - task: has-latest-opslevel-go
+      - which jq > /dev/null
```

Comment:
is this because it errors if not present?

Reply:
yes